src/components/one-time-donation/components/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,68 @@
+import { useField } from 'formik'
+import { useTranslation } from 'next-i18next'
+import {
+  FormControl,
+  FormHelperText,
+  Grid,
+  GridProps,
+  RadioGroup,
+  RadioGroupProps,
+  Typography,
+} from '@mui/material'
+import { TranslatableField, translateError } from 'common/form/validation'
+import PriceRadioButton from './RadioButton'
+
+export type RadioButtonGroup = {
+  name: string
+  options: {
+    label: string
+    value: string | number
+  }[]
+  muiRadioGroupProps?: Partial<RadioGroupProps>
+  /**
+   * Props that get passed to the `<Grid>` that wraps around each rendered button
+   * @example
+   * <Grid {...propsYouPassed}>
+   *  <RadioButton>
+   * <Grid />
+   */
```

Comment:
@kachar  What do you think about putting JSDocs on all of the common components is it worth the work?

Reply:
Nice idea, it will give good insight on how they are supposed to be used around